src/backend/joanie/tests/core/test_api_admin_course_runs.py

```diff
@@ -276,7 +276,8 @@ def test_admin_api_course_run_read_list_authenticated_with_nested_course_filter_
         """
         admin = factories.UserFactory(is_staff=True, is_superuser=True)
         self.client.login(username=admin.username, password="password")
-        now = datetime.utcnow()
+        now = datetime.now(tz=timezone.utc)
```

Comment:
use django.utils.timezone for timezone aware now?

Reply:
I used django.utils.timezone and just got :
`RemovedInDjango50Warning: The django.utils.timezone.utc alias is deprecated. Please update your code to use datetime.timezone.utc instead`

Reply:
But you can use `django.utils.timezone.now()`